LEAF_Request_Portal/admin/index.php

```diff
@@ -492,7 +492,6 @@ function hasDevConsoleAccess($login, $db_phonebook)
 $t_menu->assign('action', $action);
 $t_menu->assign('orgchartPath', Config::$orgchartPath);
 $t_menu->assign('name', XSSHelpers::sanitizeHTML($login->getName()));
-$t_menu->assign('siteType', XSSHelpers::xscrub($settings['siteType']));
```

Comment:
Does this need to be removed? It just got added back here:
https://github.com/VHAINNOVATIONS/LEAF/pull/1109/files

Reply:
no we want that line in, added it back.